src/main/java/baseball/game/SingleGame.java

```diff
@@ -0,0 +1,32 @@
+package baseball.game;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SingleGame {
+    List<Integer> computer = new ArrayList<>();
+
+    public SingleGame() {
+        while(computer.size() < 3) {
+            int randomNumber = Randoms.pickNumberInRange(1, 9);
+            if (!computer.contains(randomNumber))
+                computer.add(randomNumber);
+        }
+    }
+
+    public void setComputer (List<Integer> list) {
+        computer = list;
+    }
+
+    public GameStatus singleTurn(List<Integer> player){
+        int strike = 0, ball = 0;
+        for (int i = 0; i < player.size(); i++) {
+            if (computer.get(i) == player.get(i)) strike ++;
+            else if (computer.contains(player.get(i))) ball ++;
```

Comment:
단항연산자 앞에 공백은 삭제해주시면 가독성이 더 좋아질 것 같습니다!
```suggestion
            if (computer.get(i) == player.get(i)) strike++;
            else if (computer.contains(player.get(i))) ball++;
```